src/test/java/games/strategy/triplea/delegate/WW2V341Test.java

```diff
@@ -95,7 +95,7 @@
 import games.strategy.util.IntegerMap;
 import games.strategy.util.Match;
 
-public class WW2V3_41_Test {
+public class WW2V341Test {
```

Comment:
I'm puzzled as to why this isn't triggering a Checkstyle abbreviation violation.  Oh well. :smile:

The "V341" kind of runs together here.  I don't think there's any danger of someone not being able to parse this mentally, but naming these types of tests something like `WorldWar2Version3Year1941Test` might be easier on the eyes (if just a bit verbose).